File: Paylocity/Models/Benefits/Employee.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Paylocity.Api.Models.Benefits
{
    public class Member
    {
        public string Name { get; set; }
        public bool IsEmployee { get; set; }
    }

    public class Employee
    {
        public Member Member { get; set; }

        public List<Member> Dependants { get; set; } = new List<Member>();

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }

    public class BenefitCost
    {
        public double AnnualCost { get; set; }
        public double MonthlyCost { get; set; }
        public double CycleCost { get; set; }
        public double DiscountAmount { get; set; }
        public double PayCycles { get; set; }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }

}

File: Paylocity.UI/Application/models/Employee.js
var Employee = function() {
    var self = this;

    self.Name = ko.observable("");
    self.Dependants = ko.observableArray([]);
    self.BenefitCost = new BenefitCost();
};

var Dependant = function() {
    var self = this;

    self.Name = ko.observable("");
    self.BenefitCost = new BenefitCost();
};

var BenefitCost = function() {
    var self = this;
    
    self.AnnualCost = ko.observable(0);
    self.CycleCost = ko.observable(0);
    self.DiscountAmount = ko.observable(0);
    self.MonthlyCost = ko.observable(0);
};